### PR TITLE
Add Zillow recap post

### PR DESCRIPTION
**Pull request by bitsondatadev**

### CODE REVIEW

File: _posts/2022-11-23-trino-summit-2022-zillow-recap.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: "Optimizing Trino using spot instances with Zillow"
+author: "Santhosh Venkatraman, Rupesh Kumar Perugu, Brian Olsen"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-summit-2022/zillow.jpg
+---
+
+In the next installment of [the Trino Summit 2022 sessions posts]({% post_url
```
Comment (mosabua) [inline _posts/2022-11-23-trino-summit-2022-zillow-recap.md:9]: In this installment

File: _posts/2022-11-23-trino-summit-2022-zillow-recap.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+2022-11-21-trino-summit-2022-recap %}), we jump into an exciting topic by folks
+from Zillow about running Trino on spot instances. Spot instances are cheap and
```
Comment (mosabua) [inline _posts/2022-11-23-trino-summit-2022-zillow-recap.md:11]: Link to Zillow website?

File: _posts/2022-11-23-trino-summit-2022-zillow-recap.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+ephemeral nodes that lead to much cheaper compute. Spot instances are cheaper as
```
Comment (mosabua) [inline _posts/2022-11-23-trino-summit-2022-zillow-recap.md:12]: lead to reduced overall compute costs.

File: _posts/2022-11-23-trino-summit-2022-zillow-recap.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+Zillow's BI platform team is tasked with enabling access to data and metrics
+from their data lake in a self-serving and performant manner. The platform must
+handle generating up-to-date reports and metrics to unlock time-critical
+opportunities. They also need to enable adhoc analytics across multiple domains
+within Zillow. 
+
+There are close to 600 data pipelines and 65,000 queries running daily. The
+average read covers 600 terabytes of data, and the average P95 time is around
+20 seconds. They have 6 Trino services that service various workflow based on
```
Comment (mosabua) [inline _posts/2022-11-23-trino-summit-2022-zillow-recap.md:36]: six Trino clusters


Or what is a "Trino service"

File: _posts/2022-11-23-trino-summit-2022-zillow-recap.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+load. These are all deployed on Amazon EKS with a range of eight to 60 workers
+based on CPU utilization.
+
+When deploying Trino on EKS, Zillow uses worker groups, which enables them to
+collocate nodes in AWS local zones. It also made it possible to choose spot 
+instances, which are 90% cheaper than regular on-demand instances. A critical
+aspect they needed to do was correctly tune the percentage of nodes that were
```
Comment (mosabua) [inline _posts/2022-11-23-trino-summit-2022-zillow-recap.md:43]: needed to cover was to correctly tune

File: _posts/2022-11-23-trino-summit-2022-zillow-recap.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+spot instances, and create pools of nodes that were entirely on-demand
+(coordinators) and some that are an optimal blend of spot and on-demand.
```
Comment (mosabua) [inline _posts/2022-11-23-trino-summit-2022-zillow-recap.md:45]: on-demand for coordinators

Comment (mosabua) [reply _posts/2022-11-23-trino-summit-2022-zillow-recap.md:45]: also .. we do we need to explain why? Coordinator being spot would take the whole cluster down

**Review by mosabua — CHANGES_REQUESTED** [review]

File: _posts/2022-11-23-trino-summit-2022-zillow-recap.md (hunk 4 above)
Comment (bitsondatadev) [reply _posts/2022-11-23-trino-summit-2022-zillow-recap.md:36]: It's likely better called a "cluster" but it's Zillow's abstraction..could be multiple clusters behind a proxy

File: _posts/2022-11-23-trino-summit-2022-zillow-recap.md [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+
+Zillow's BI platform team is tasked with enabling access to data and metrics
+from their data lake in a self-serving and performant manner. The platform must
+handle generating up-to-date reports and metrics to unlock time-critical
+opportunities. They also need to enable adhoc analytics across multiple domains
+within Zillow. 
+
+There are close to 600 data pipelines and 65,000 queries running daily. The
+average read covers 600 terabytes of data, and the average P95 time is around
+20 seconds. They have six Trino clusters that service various workflow based on
```
Comment (mosabua) [inline _posts/2022-11-23-trino-summit-2022-zillow-recap.md:37]: workflows ?

**Review by mosabua — APPROVED** [review]
Looks good to go.

File: _posts/2022-11-23-trino-summit-2022-zillow-recap.md (hunk 7 above)
Comment (bitsondatadev) [reply _posts/2022-11-23-trino-summit-2022-zillow-recap.md:37]: yup I'll fix

**Pull request merged**